<?xml version="1.0"?>
<launch>

    <!--  Arm Controllers -->
  <rosparam command="load" file="$(find pr2_mannequin_mode)/config/pr2_arm_controllers_loose.yaml" />
  <node pkg="pr2_controller_manager" type="spawner" name="spawn_arm_controllers"
        args="--stopped r_arm_controller_loose l_arm_controller_loose" />

  <!--  Left Arm Mannequin Mode -->
  <group ns="l_arm_controller_loose">
   <node type="trajectory_lock.py"
         pkg="pr2_mannequin_mode"
         name="traj_lock"
         output="log"
         args=".02 .02 .02 .02 .02 .06 .06" />
  </group>

  <!--  Right Arm Mannequin Mode -->
  <group ns="r_arm_controller_loose">
   <node type="trajectory_lock.py"
         pkg="pr2_mannequin_mode"
         name="traj_lock"
         output="log"
         args=".02 .02 .02 .02 .02 .06 .06" />
  </group>

   <node name="art_basic_control" pkg="art_basic_control"
	type="node.py" output="screen" respawn="true">

  </node>

</launch>
